qiskit_ibm/api/rest/runtime.py

```diff
@@ -148,7 +150,8 @@ def program_run(
             'group': group,
             'project': project,
             'backend': backend_name,
-            'params': [params]
+            'params': [params],
+            'runtime': runtime
```

Comment:
The default of `runtime` in `IBMRuntimeService.run()` is empty string. Wouldn't this make the request fail due to invalid image format?

Reply:
The ntc side will skip image format validation if `runtime` is empty string.